docs/3 Scaffold a Blockchain/1 Introduction.md

```diff
@@ -0,0 +1,11 @@
+# Scaffold a Blockchain
+
+<!-- what is the general message we want to give? intro to the structure that is created for project and types or? we should say something about why we specify a github repo -->
+
+The `starport app` command scaffolds a project.
```

Comment:
```suggestion
The project directory of any Cosmos SDK blockchain contains many directories, source code files, configuration files, scripts, etc. Some of these files implement custom logic and are very specific to a particular project. Other files, however, are common between different Cosmos SDK projects and act as wiring between different parts of the project. Starport CLI automatically generates this common (boilerplate) code and helps in scaffolding custom functionality, so that developers can focus on application-specific logic.

One of the core features of Starport CLI is code scaffolding.
```